design/help_overview.md

```diff
@@ -6,21 +6,21 @@ pywbemcli  --help
+b"Usage: pywbemcli class [COMMAND-OPTIONS] COMMAND [ARGS]...\n\n  Command Group to manage CIM Classes.\n\n  In addition to the command-specific options shown in this help text, the\n  general options (see 'pywbemcli --help') can also be specified before the\n  command. These are NOT retained after the command is executed.\n\nOptions:\n  --help  Show this message and exit.\n\nCommands:\n  associators   Get the associated classes for the CLASSNAME.\n  enumerate     Enumerate classes from the WBEMServer.\n  find          Find all classes that match the CLASSNAME...\n  get           get and display a single CIM class from the...\n  hierarchy     Display class inheritance hierarchy as a...\n  invokemethod  Invoke the class method named methodname in...\n  names         Get and display classnames from the...\n  references    Get the reference classes for the CLASSNAME.\n"
 ```
 
 pywbemcli class get --help
 ==========================
 
 ```
-b'Usage: pywbemcli class get [COMMAND-OPTIONS] CLASSNAME\n\n  get and display a single class from the WBEM Server\n\nOptions:\n  -l, --localonly                 Show only local properties of the class.\n  --includequalifiers / --no_includequalifiers\n                                  Include qualifiers in the result. Default is\n                                  to include qualifiers\n  -c, --includeclassorigin        Include classorigin in the result.\n  -p, --propertylist <property name>\n                                  Define a propertylist for the request. If\n                                  not included a Null property list is defined\n                                  and the server returns all properties. If\n                                  defined as empty string the server returns\n                                  no properties. ex: -p propertyname1 -p\n                                  propertyname2 or -p\n                                  propertyname1,propertyname2\n  -n, --namespace <name>          Namespace to use for this operation. If\n                                  defined that namespace overrides the general\n                                  options namespace\n  --help                          Show this message and exit.\n'
```

Comment:
This results in a very long line, not only in the source file but also in the rendered output. Split accordingly.